### PR TITLE
Fix alerts

### PR DESCRIPTION
**Pull request by zathras-crypto**
Alerts got PR'd with a broken packet check - sorry about that - here is the fix

### CODE REVIEW

**Comment by dexX7** [comment]
Yes, I noticed this, too (thus "this isn't working" in the other thread). But maybe it's better to leave it this way or to change the whitelisted addresses, so not everyone is bothered with the "Tx 99 ..." warning.

**Comment by zathras-crypto** [comment]
Thanks DexX :) - the testing was still active, fixed.

File: src/rpcmisc.cpp [hunk 1]
```diff
@@ -60,7 +60,6 @@ Value getinfo(const Array& params, bool fHelp)
     GetProxy(NET_IPV4, proxy);
 
     Object obj;
-    obj.push_back(Pair("mastercoreversion",  (int)1131008));  // strip the initial '10' : for tagged release go to 10007, etc.
```
Comment (m21) [inline src/rpcmisc.cpp:63]: Uh-oh.
Why delete 'mastercoreversion' ?

Comment (dexX7) [inline src/rpcmisc.cpp:63]: https://github.com/mastercoin-MSC/mastercore/pull/207/files#r20734476

Imho there are two options:
- show a version in both calls (getinfo, getinfo_MP) **that are equal**
- show a version in either one of the calls

Consistency is key.

**Comment by zathras-crypto** [comment]
So for clarity on this one...

getinfo as an RPC call is a bitcoin core call, not a master core call, and it returns bitcoin related information.  Initially we simply hijacked this to drop in a version.

Now we have other info to report, I've placed in a getinfo_MP call.  So just as we have done with other calls (eg bitcoin core gettransaction to master core gettransaction_MP) we have done the same with the information related to master core.  The version is also now more usable for version checks per alerts.

TL:DR; Bitcoin Core info is in getinfo, Master Core info (including version) is in getinfo_MP.

Thanks :)
